pkg/cmd/kind/version/version_test.go

```diff
@@ -60,3 +60,59 @@ func TestTruncate(t *testing.T) {
 		})
 	}
 }
+
+func TestVersion_Success(t *testing.T) {
```

Comment:
I would call this `TestVersion` since that's what it does and there is no error path, but I'm not going to block the PR on that

Reply:
updated